staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -225,12 +226,13 @@ func RecordLongRunning(req *http.Request, requestInfo *request.RequestInfo, comp
 // a request. verb must be uppercase to be backwards compatible with existing monitoring tooling.
 func MonitorRequest(req *http.Request, verb, group, version, resource, subresource, scope, component, contentType string, httpCode, respSize int, elapsed time.Duration) {
 	reportedVerb := cleanVerb(verb, req)
+	dryRun := cleanDryRun(req.URL.Query()["dryRun"])
```

Comment:
If I was being malicious, could I pass in a ton of different dryRun values here and explode the carnality of the metrics? Maybe validate dryRun to constrain it to an allowed set of values?